rs/index_server/src/index_server.rs

```diff
@@ -119,30 +119,47 @@ impl IndexServer for IndexServerImpl {
         if let Some(max_time_to_flush_ms) = req.max_time_to_flush_ms {
             collection_config.max_time_to_flush_ms = max_time_to_flush_ms as u64;
         }
+        if collection_config.use_distributed_log_as_wal {
+            collection_config
+                .topic_name
+                .push_str(&format!("-{collection_name}"))
+        }
```

Comment:
you'll need to add this to muopdb.proto and follow the convention similar to other fields

Reply:
why do we need to append the `-collection_name` at the end?